### DISCUSSION TITLE
Hold soft reset cycle start message

### DISCUSSION THREAD

**Discussion by William-i-am** [winder/Universal-G-Code-Sender#2398] [Help]
How do I clear this message "**Hold soft reset cycle start message**"
Windows version
*** Connected to GRBL 2.1
Grbl 1.1f ['$' for help]
Grbl for GD32
Version:GD32 V2.1

![Hold soft reset cycle start message](https://github.com/winder/Universal-G-Code-Sender/assets/13262758/20d47a54-48fe-4ba8-bb44-0fa1f110d503)

Thank you for the read.

**Comment by breiler** [comment]
Your controller is reporting that the input pins for "HOLD", "SOFT RESET" and "CYCLE START" are activated. You need to either check that those inputs are properly connected or invert them in your firmware config. Ask the company that you bought the controller.
